packages/asak/asak.0.4/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+
+synopsis: "Partitioning OCaml codes"
+description:
+"""
+Asak provides functions to parse, type-check and identify similar OCaml codes.
+These functions are then used to partition codes implementing the same function and help to detect code duplication.
+"""
+
+maintainer: ["alexandre@moine.me"]
+authors: ["Alexandre Moine"]
+license: "MIT"
+
+homepage: "https://github.com/nobrakal/asak"
+doc: "https://nobrakal.github.io/asak/asak/index.html"
+bug-reports: "https://github.com/nobrakal/asak/issues"
+dev-repo: "git+https://github.com/nobrakal/asak.git"
+
+depends: [
+  "ocaml"    {>= "4.05" & < "4.15"}
+  "cmdliner" {>= "1.0.0"}
```

Comment:
```suggestion
  "cmdliner" {>= "1.1.0"}
```
For 

    # File "bin/anzad.ml", line 199, characters 2-10:
    # Error: Unbound module Cmd